### ISSUE TITLE
[fvt]2.13.5 rvitals node power should not return "Error: Energy data requires..." message

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#3124]
env:Bmc env
build:[root@fs1 ~]# lsxcatd -v
Version 2.13.4 (git commit 305ec7d71e4333f43f325e546daaf97fa943cd56, built Wed May 17 06:15:47 EDT 2017)
[root@fs1 ~]# 
How to reproduce:
```
1.Node defintion:
[root@fs1 ~]# lsdef frame45cn07
Object name: frame45cn07
    arch=ppc64le
    bmc=50.45.7.1
    bmcvlantag=551
    chain=runcmd=date,osimage=rhels7.3-GA-ppc64le-netboot-compute
    cons=ipmi
    cpucount=160
    cputype=POWER8 (raw), altivec supported
    currchain=osimage=rhels7.3-GA-ppc64le-netboot-compute
    currstate=osimage=rhels7.3-GA-ppc64le-netboot-compute
    disksize=sda:1000GB,sdb:1000GB
    groups=all,disco,compute,frame45,firestone
    installnic=mac
    ip=192.168.45.7
    mac=98:be:94:59:ea:27
    memory=261482MB
    mgt=ipmi
    mtm=8335-GTA
    netboot=petitboot
    nodetype=mp
    os=rhels7.3-GA
    postbootscripts=otherpkgs
    postscripts=syslog,remoteshell,syncfiles
    profile=compute
    provmethod=rhels7.3-GA-ppc64le-netboot-compute
    serial=1210532
    status=success updating firmware
    statustime=04-12-2017 06:14:31
    supportedarchs=ppc64
    switch=frame45sw1
    switchport=7
[root@fs1 ~]# 

2
[root@fs1 ~]# rvitals  frame45cn07 power
frame45cn07: Error: :Energy data requires additional IBM::EnergyManager plugin in conjunction with IMM managed IBM equipment
-----------------> should not return Error:Energe ..
frame45cn07: CPU VDD Curr: 58 Amps
frame45cn07: Power Status: on
[root@fs1 ~]# echo $?
1
---------------->  Power status has output so the command's return value should not be 1
```

**Comment by gurevichmark** [comment]
@junxiawang Not sure about the "openbmc" label for this issue. The `frame45cn07` is `ipmi` managed

**Comment by junxiawang** [comment]
@gurevichmark I have no openbmc machine to test the command and I do not know whether openbmc machine has such error so I add openbmc label here.

**Comment by gurevichmark** [comment]
Ok, but this issue seems to be still valid for `ipmi` managed machines - why do we return an error and RC perhaps should be 0.

As for `openbmc` managed machines, it is still not clear what kind of data will be returned by `rvitals power`. Once OpenBMC team implements that sensor we will have a better idea what kind of data is returned.

**Comment by whowutwut** [comment]
@gurevichmark I completed the rvitals based on the API PR #3134,  but even with that code change, we are not invoking another plugin, so the message does not apply for openbmc.  
  
But I agree with @junxiawang , we probably should not print that error message if we can tell what machine we are going against. . We do not support that for OpenPower (IPMI) .

**Comment by zet809** [comment]
Yes, @whowutwut , but we shall not output that message for OpenPOWER.
And, every for x86_64, I think we shall also not output that message if "IBM::EnergyManager" is not installed.

**Comment by gurevichmark** [comment]
Removing "component: openbmc" label since this issue should be fixed for OpenPower machines.
The code path for OpenBmc will be completely different for `rvitals power`.

**Comment by whowutwut** [comment]
@zet809  Yes, agreed, would be a easy fix to not print out a error message.

**Comment by zet809** [comment]
Have been fixed in PR #3546 , so close this issue.
